source/_integrations/linear_garage_door.markdown

```diff
@@ -22,3 +24,7 @@ The Linear integration lets you control Linear garage doors through Home Assista
 ## Cover
 
 Garage doors linked to your Linear account will appear as covers.
+
+## Light
+
+Lights on your garage door will appear as lights.
```

Comment:
For future content, for me, it would be helpful to have a bit more context in the documentation. What can it do? Just turn on and off? Does it have brightness, color control, etc.